src/pypulseq/make_arbitrary_rf.py

```diff
@@ -63,8 +67,14 @@ def make_arbitrary_rf(
         System limits.
     time_bw_product : float, default=4
         Time-bandwidth product.
-    use : str, default=str()
+    use : str, default='undefined'
         Use of arbitrary radio-frequency pulse event. Must be one of 'excitation', 'refocusing' or 'inversion'.
```

Comment:
The default is not listed as valid option. The valid options listed in get_supported_rf_uses() are:
'excitation', 'refocusing', 'inversion', 'saturation', 'preparation', 'other', 'undefined'